### PR TITLE
Fix websocket close handshake issue and race condition when websocket…

### PR DESCRIPTION
**Pull request by pan-wang**
… client disconnect without close handshake #77

### CODE REVIEW

File: src/AspNetCore/Src/aspnetcoreconfig.cxx [hunk 1]
```diff
@@ -376,6 +376,7 @@ ASPNETCORE_CONFIG::Populate(
         strExpandedEnvValue.Reset();
         pEnvVar->Release();
         pEnvVar = NULL;
+        pEntry->Dereference();
```
Comment (jkotalik) [inline src/AspNetCore/Src/aspnetcoreconfig.cxx:379]: Is this related to the fix or is it unrelated.

Comment (pan-wang) [reply src/AspNetCore/Src/aspnetcoreconfig.cxx:379]: this fixes a memory leak

**Review by jhkimnew — APPROVED** [review]
Looks good to me.

**Comment by pan-wang** [comment]
thanks

**Comment by risogolo** [comment]
So, when this bugfix will be released as a part of new version of nuget package for websockets?

**Comment by Tratcher** [comment]
@risogolo the bugfix is not in the websocket nuget package, it's in the AspNetCoreModule which installs via MSI. Expect the MSI with 2.1.0-preview.

**Comment by risogolo** [comment]
but this bug is also in 1.x

**Comment by Tratcher** [comment]
ANCM does not version like packages, it's an in place update. The next MSI released will affect all versions of ASP.NET Core.

**Pull request merged**